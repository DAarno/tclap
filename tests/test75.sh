#!/bin/sh
# failure  validates that the correct error message
# is displayed for XOR'd args
`dirname $0`/simple-test.sh `basename $0 .sh` test20 -b -a
