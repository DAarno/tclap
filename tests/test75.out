PARSE ERROR: Argument: -b AND -a provided.
             Only one is allowed.

Brief USAGE: 
   ../examples/test20  {-a|-b} [--] [--version] [-h]

For complete USAGE and HELP type: 
   ../examples/test20 --help

